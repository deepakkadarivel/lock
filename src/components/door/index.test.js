import React from 'react';
import { create } from 'react-test-renderer';
import Door from './index';

describe('Door component', () => {
    test('Renders Door component', () => {
        const component = create(<Door text="door name" onClick={() => {}} />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
