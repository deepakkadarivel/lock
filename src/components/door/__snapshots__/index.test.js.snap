// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Door component Renders Door component 1`] = `
<div
  className="Doors-item"
  onClick={[Function]}
  role="presentation"
>
  <i
    className="material-icons"
  >
    meeting_room
  </i>
  door name
</div>
`;
